src/Components/GoogleAnalytics.js

```diff
@@ -37,7 +38,10 @@ class GoogleAnalytics extends React.Component {
   }
 
   render() {
-    if (!this.state.trackingId) {
+    if (
+      !this.state.trackingId ||
+      !(getCookieValue("CookieConsent") === "true")
```

Comment:
Let's create a more high level method for the cookie consent. How about `cookieConsentGiven()`?

It could looks something like this (not yet tested):

```js
import { Cookies } from "react-cookie-consent";

export default function cookieConsentGiven() {
  return Cookies.get("CookieConsent") === "true";
}
```

If so we can simply remove the `getCookieValue` method.